rollastic/node.py

```diff
@@ -54,17 +55,26 @@ def populate(self):
         self.clear()
 
         info = self.cluster.es.nodes.info(self.node_id)['nodes'].get(self.node_id, {})
+        isbad = False
```

Comment:
try:
   info = self.cluster.es.nodes.info(self.node_id)['nodes'].get(self.node_id, {})
   if not info:
      raise Exception('Got bad result attempting to retrieve node info. node_id={} info={}')...
  self.update(info) 
  stats = self.cluster.es.nodes.stats(self.node_id)['nodes'].get(self.node_id, {})
      raise Exception('Got bad result attempting to retrieve node stats. ...)
  self.update(stats)
  self.unsuccessful_retrieval_attempts = 0 # do we even care to do this? once we update we're done with the node, right?
except Exception as e:
   self.unsuccessful_retrieval_attempts += 1
   if self.unsuccessful_retrieval_attempts > 10:
      raise Exception(''Reached retrieval limit for node...)